src/Abc.Zebus.Tests/Directory/PeerDirectoryClientTests.cs

```diff
@@ -26,6 +30,58 @@ public partial class PeerDirectoryClientTests
         private TestBus _bus;
         private Peer _self;
         private Peer _otherPeer;
+        private Mock<IMessageDispatcher> _messageDispatcher;
+        private MessageDispatch _dispatchedMessage;
+
+        class MySnap : IEvent
+        {
+        }
+
+        class MyMessage : IEvent
+        {
+        }
+
+        class MyMessageHandlerInvoker : MessageHandlerInvoker
+        {
+            public MyMessageHandlerInvoker()
+                : base(typeof(MySubscriptionHandler), typeof(SubscriptionUpdatedMessage))
+            {
+            }
+
+            public override void InvokeMessageHandler(IMessageHandlerInvocation invocation)
+            {
+                throw new NotImplementedException();
```

Comment:
`NotImplementedException` shows as a TODO item in the IDE, please use something else (also for the other cases below)